js/script.js

```diff
@@ -1,63 +1,85 @@
-var NAV_BLOCK_ID = "#nav";
 var NAV_LIST_ITEM_SELECTOR = ".nav-list";
 var CONTACTS_SELECTOR = ".contact-icon";
 var POPUP_SELECTOR = ".popup";
+var NAV_BLOCK_ID = "#nav";
 var JOIN_BUTTON_ID = "#join"
 var EMAIL_INPUT_ID = "#your-mail";
-var SLIDER_PICTURE_ONE_ID = "#slider-picture1"
-var SLIDER_PICTURE_TWO_ID = "#slider-picture2"
-var SLIDER_BUTTON_ID = "#slider-button"
+var SLIDER_PICTURE_ONE_ID = "#slider-picture1";
+var SLIDER_PICTURE_TWO_ID = "#slider-picture2";
+var SLIDER_BUTTON_ID = "#slider-button";
+var NAV_ACTIVE_CLASS = "nav-active";
+var CONTACT_ACTIVE_CLASS = "active";
+var POPUP_TEXT_ATTR = "data-popuptext";
+
+var navPosition;
+var navMenu;
+var navList;
+var navMenuHeight;
 
 $(document).ready( function() {
+
   navMenu = $(NAV_BLOCK_ID);
+  navMenuHeight = navMenu.outerHeight();
   navPosition = navMenu.offset().top;
   navList = $(NAV_LIST_ITEM_SELECTOR)
-  fixedNav();
+
+  positionToFixed();
+
+  window.onscroll = positionToFixed;
+
   navList.click(navScroll);
   
   $(CONTACTS_SELECTOR).click(function() {
     var cont = $(CONTACTS_SELECTOR);
```

Comment:
Is it an array? If it - use contactsArray or similar. If not - use full word 'contact'.